gitlab/pipeline.ml

```diff
@@ -136,6 +148,9 @@ let set_active_repos ~(installation : Installation.t Current.t)
        (fun acc r -> Index.Repo_set.add r.Gitlab.Repo_id.name acc)
        Index.Repo_set.empty
   |> Index.set_active_repos ~owner:installation.Installation.name;
+  Prometheus.Gauge.set
+    (Metrics.repositories_total installation.Installation.name)
+    (float_of_int (List.length repos));
```

Comment:
Shouldn't this setting system be hidden into the metrics, so we don't have to worry about the type? Like:
```ocaml
Metrics.set_repositories_total installation.Installation.name (float_of_int (List.length repos));
```
This way we have more control over the metrics inside the `Metrics` module. WDYT?

Reply:
Sure, when this gets moved into the ocurrent plugin.